010-provider-architecture/002-final/lib/core/models/user.dart

```diff
@@ -1,25 +1,20 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'user.g.dart';
+
+@JsonSerializable()
```

Comment:
I've never used this, but this is awesome! thanks for showing me this, I'll write a tutorial on it asap. I don't think a lot of Flutter developers know about this.